migrations/20200605172945_add_commit_score_column.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table("commit", (builder) => {
+    builder.integer("score").notNullable();
+  });
+}
+
+export async function down(knex: Knex): Promise<unknown> {
+  const exists = await knex.schema.hasColumn("commit", "score");
+  if (exists) {
```

Comment:
We don't need this check. We can assume the migration is already applied when the down migration is executing, i.e. score exists.